tests/integ/scala/test_update_delete_merge_suite.py

```diff
@@ -698,6 +699,78 @@ def test_snow_1694649_repro_merge_with_equal_null(session):
     ]
 
 
+def test_update_with_variant_type(session):
```

Comment:
just curious, do we have a test that verify other data type can be handled?

Reply:
yes, other tests are testing the non-variant types